about.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poiret+One&family=Poppins:wght@300&display=swap" rel="stylesheet"> 
+    <link href="stylesheets/style.css" rel="stylesheet" />
+    <link href="stylesheets/about.css" rel="stylesheet" />
+    <title>Shayla Logan</title>
+  </head>
+
+  <body class="container">
+    
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <a class="navlink" href="index.html">Home</a>
+            <a class="navlink" id="active">About</a>
+            <a class="navlink" href="portfolio.html">Projects</a>
+            <a class="navlink" href="contact.html">Contact</a>
+          </li>
+        </ul>
+      </nav>
+
+      <h1>About</h1>
+    </header>
+
+    <main>
+      <section class="content">        
+        <h2>Hi there! I'm Shayla Logan.</h2>
+        <p>
+          I started in the tech indsutry as a mainframe programmer. Later, I spent over 15 years as a software tester and Business Analyst before transitioning back into coding.
+        </p>
+        <p>  
+          I love working as a web developer and seeing the impact a website can have on the world around me.
+        </p>
+        <p>
+          In my spare time I can be found woodworking, cooking, listening to a wide variety of music, watching psychological-thriller movies, and cuddling with my two dogs.
+        </p>            
```

Comment:
If you want to create breaks in a large p tag you can use the `<br>` tag or you can use `<pre>` it preserves spacing and line breaks. More about it is here: https://www.w3schools.com/html/tryit.asp?filename=tryhtml_pre